src/xopen/__init__.py

```diff
@@ -358,6 +366,11 @@ def _raise_if_error(self, check_allowed_code_and_message: bool = False,
         """
         retcode = self.process.poll()
 
+        if sys.platform.startswith("win") and retcode == 1 and stderr_message == b'':
+            # Special case for Windows. Winapi terminates processes with exit code 1
+            # and an empty error message.
+            return
```

Comment:
@marcelm Can you have a look at this? 
The following is the case.
The winapi terminates processes with exit code one by default, and not with signal.SIGTERM. (This is a bug IMO, but what can you do about it?). 

If python -m isal.igzip crashes on windows it will also give exit code 1. (Only external application readily available on windows). Luckily that will *always* fail verbosely, due to the nature of the python interpreter. By checking for platform,  exit code and empty error message I think we can be quite sure it works correctly.

However this has to be implemented outside the "_allowed_exit_code" and "_allowed_error_message" checks in order to function correctly.

Reply:
Ah, that was the last remaining thing that I didn’t have time to figure out, thanks for that. (Probably I wouldn’t have gone through the trouble of installing PyCharm in my Windows VM, so it would have taken me much longer.)

Yes, let’s handle it the way you did it.

I’ll change `sys.platform.startswith("win")` to `sys.platform == "win32"`, however, because that’s how we wrote it in other parts of the code. This is fine because `sys.platform` is guaranteed to be "win32" even on a 64-bit Windows.